allo/ir/unify_kernels.py

```diff
@@ -0,0 +1,22 @@
+# Copyright Allo authors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=no-name-in-module
+
+from collections.abc import Callable
+
+from .._mlir.ir import (
+    Context,
+)
+from .._mlir.dialects import (
+    allo as allo_d,
+)
+from ..customize import customize
+
+
+def unify_kernels(func1: Callable, func2: Callable, loop_num: int):
```

Comment:
Add docstring for this function to explain its functionality. It is important to explain what `loop_num` is.